nixos/modules/services/x11/xserver.nix

```diff
@@ -409,6 +409,16 @@ in
           if possible.
         '';
       };
+
+      enableCtrlAltBackspace = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable the DontZap option, which binds Ctrl+Alt+Backspace
```

Comment:
I would personally avoid negations in option names, so the current or `enableZap` are better IMHO than `enableDontZap`.